worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -553,8 +553,7 @@ public void save(
         AsyncCommandBuilder.wrap(task, actor)
                 .registerWithSupervisor(worldEdit.getSupervisor(), "Saving schematic " + filename)
                 .setDelayMessage(Caption.of("worldedit.schematic.save.saving"))
-                .onSuccess(Caption.of("fawe.worldedit.schematic.schematic.saved", filename, (overwrite ?
-            " (overwriting previous file)." : ".")), null)
+                .onSuccess(Caption.of("fawe.worldedit.schematic.schematic.saved", filename, (overwrite ? Caption.of("fawe.worldedit.schematic.schematic.overwritten") : ".")), null)
```

Comment:
Sorry, I meant two entirely separate captions, we shouldn't be inserting a caption in here at all. Also, I believe this line will be too long. Please see the .editorconfig in the project root